app/helpers/item_seasons_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module ItemSeasonsHelper
+  def rotation(month_index)
+    case month_index
+    when 1, 6
+      'rotate-plus-5'
+    when 5
+      'rotate-minus-5'
+    when 12
+      'rotate-plus-10'
+    when 4, 9
+      'rotate-minus-10'
+    when 2, 3, 7, 8, 10, 11
+      'rotate-0'
+    else
+      raise 'month-index must be between 1 and 12'
+    end
+  end
```

Comment:
**suggestion:** I think using guard clauses would be easier to read:

```suggestion
  def rotation(month_index)
    raise 'month-index must be between 1 and 12' unless 1..12.include?(month_index)
    
    case month_index
    when 1, 6
      'rotate-plus-5'
    when 5
      'rotate-minus-5'
    when 12
      'rotate-plus-10'
    when 4, 9
      'rotate-minus-10'
    else
      'rotate-0'
    end
  end
```

Reply:
I accepted this change, but it wasn't passing the tests, and rubocop didn't like it. So I changed it.
Question: what is the benefit of using a guard clause here instead of how it was written before? Is the fact that it runs before anything the advantage?

Reply:
On benefit: it's easier to read IMO. Putting what you expect as arguments as the very first thing sets clear rules for arguments. Performance-wise there is a bit of difference but it's negligible.

Now on Rubocop and test: what breaks and how does it break?

Reply:
As it was written before: `raise 'month-index must be between 1 and 12' unless 1..12.include?(month_index)`

All the helper tests didn't pass, except for the last ones:

<img width="868" alt="Screen Shot 2022-09-19 at 15 23 42" src="https://user-images.githubusercontent.com/87862340/191087854-6cbbf803-fa86-44cc-bf77-47ee28545f61.png">

As for rubocop, we got: `Avoid the use of flip-flop operators`

But changing it to `raise 'month-index must be between 1 and 12' if month_index > 12 || month_index < 1` seems to have fixed our problems. Is this ok?

Reply:
aaaaaaaah! Now I get it 😄 . I was wondering why it wasn't working, I didn't know what a flip flop operator was, so I rewrote it 😬 Since it's only a matter of taste, I'm leaving `raise 'month-index must be between 1 and 12' if month_index > 12 || month_index < 1`